### DISCUSSION TITLE
Tell me more about Decision backlog?

### DISCUSSION THREAD

**Discussion by Relequestual** [thomvaill/log4brains#37] [General]
Log4Brains looks great and I look forward to trying it!

Under "coming soon", you list "Decision backlog". Can you tell me more about this feature please?

**Comment by thomvaill** [comment]
Thanks ! :-)

What I had in mind for the decision backlog was very simple: just a `backlog.md` file with bullet points of the remaining decisions to make, which would be displayed in a "Backlog" tab in the UI.

Do you have other ideas?

**Reply by Relequestual** [reply]
Cool. Sounds reasonable!
